pom.xml

```diff
@@ -49,73 +47,67 @@
   </developers>
 
   <scm>
-    <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}.git</connection>
-    <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}.git</developerConnection>
-    <url>http://github.com/jenkinsci/${project.artifactId}</url>
+    <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+    <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
     <tag>HEAD</tag>
   </scm>
 
-  <repositories>
-    <repository>
-      <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
-    </repository>
-  </repositories>
+  <properties>
+    <jenkins.version>2.138.4</jenkins.version>
```

Comment:
it's inherited from the parent pom (https://github.com/jenkinsci/plugin-pom/blob/5c6ebd4d5dff9f41e83359caf757b084dcaf2ef9/pom.xml#L52). Why put it here?

Reply:
Because if in the future we want to upgrade the core requirement but have no need to update the parent, there is little to do. It also show precisely which version of core the plugin is targeting, without having to go to the parent. It is also good to make sure we keep this and the bom in sync.

Reply:
Personally I always enjoy seeing the `jenkins.version` being fixed in the pom.xml instead of having to dig into the parent pom specific version and so on. Especially interesting when you have multiple plugins (or multiple versions of a plugin) to check/investigate